CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+1.2.8
```

Comment:
if we merge this change into master, it will become 1.3, as master is for 1.3 and there are other changes in it. do you urgently need a fixed release? vondrasekpavel is busy doing lots of cleanups and fixes, we would release sometimes in february.

for an urgent fix, i would have to update the 1.2 branch to have everything that is in 1.2 (it currently looks massively outdated: https://github.com/jackalope/jackalope-doctrine-dbal/compare/1.2...1.2.8). if you need a fix urgently, i can do that and then merge your branch into 1.2 and tag 1.2.9 instead.